packages/fast-components/src/badge/fixtures/base.html

```diff
@@ -0,0 +1,71 @@
+<fast-design-system-provider style="font-family: var(--body-font)">
+    <style>
+        fast-badge {
+            --badge-fill-primary: #D223CB;
+            --badge-fill-secondary: #3278CD;
+            --badge-fill-tertiary: #DFDFDF;
+            --badge-fill-foo: #EE1010;
+            --badge-fill-bar: rgba(238,6,16,0.7);
+            --badge-color-foo: white;
+            --badge-color-bar: black;
+        }
+    </style>
+    <h1>Badge</h1>
+    <h4>Default</h4>
+    <fast-badge fill="primary" style="font-size: 10px; line-height: 16px;" color="foo" id="foo">
+        Badge
+    </fast-badge>
+    <fast-badge fill="secondary" color="foo" id="foo">
+        Badge
+    </fast-badge>
+    <fast-badge fill="tertiary" style="font-size: 14px; line-height: 20px;" color="bar" id="foo">
+        Badge
+    </fast-badge>
+    <h4>Circular</h4>
+    <fast-badge fill="primary" color="foo" id="foo" circular="true">
+        9
+    </fast-badge>
+    <fast-badge fill="secondary" color="foo" id="foo" circular="true">
+        99
+    </fast-badge>
+    <fast-badge fill="tertiary" color="bar" id="foo" circular="true">
+        9999999
+    </fast-badge>
+    <h4>Examples</h4>
+    <div>
+        <div style="position: relative; display: inline-flex;">
+            <img src="https://placehold.it/120x140/DFDFDF/171717">
+            <fast-badge style="position: absolute; right: 4px; top: 4px;" fill="foo" color="foo" id="foo">
+                Sale
+            </fast-badge>
+        </div>
+        <div style="position: relative; display: inline-flex;">
+            <img src="https://placehold.it/140x140/DFDFDF/171717">
+            <fast-badge style="position: absolute; right: 4px; top: 4px;" fill="secondary" color="foo" id="foo">
+                App of the day
+            </fast-badge>
+        </div>
+    </div>
+    <div>
```

Comment:
Can you make some of these long repeated inline styles a css class in your style element so it reduces clutter?